channels/in.m3u

```diff
@@ -503,32 +499,126 @@ http://95.216.167.183:5080/LiveApp/streams/443106610169904881506470.m3u8
 https://rtmp.smartstream.video/velichamtv/velichamtv/playlist.m3u8
 #EXTINF:-1 tvg-id="XploreChannel.in" tvg-name="Xplore Channel" tvg-country="IN" tvg-language="" tvg-logo="" group-title="",Xplore Channel (720p)
 http://cdn18.live247stream.com/ndachannel/tv/playlist.m3u8
-#EXTINF:-1 tvg-id="ZLiving.in" tvg-name="Z Living" tvg-country="IN" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/ZLiving_512x512.png?raw=true" group-title="",Z Living
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8?OTATV
-#EXTINF:-1 tvg-id="ZLiving.in" tvg-name="Z Living" tvg-country="IN" tvg-language="" tvg-logo="https://www.zliving.com/wp-content/uploads/2018/02/Zliving-Logo.png" group-title="",Z Living
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/mono.m3u8
```

Comment:
works, long connection tho

Reply:
Link exists below

Reply:
It's also a dupe of Line 542

Reply:
this one is a prod stream, a bit different but can be a backup link

Reply:
Noted, reinstating it.